frontend/src/components/GitlabDashboard/GitlabApp.js

```diff
@@ -0,0 +1,127 @@
+import { React, useEffect, useState } from "react";
+import Profile from "./GitlabProfile";
+import { FcApproval, FcCancel } from "react-icons/fc";
+const axios = require("axios");
+
+function GitlabApp({ user }) {
```

Comment:
variable naming comments from previous MR have not been resolved. @rithikagarwal2